pkgs/desktops/deepin/library/deepin-wayland-protocols/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deepin-wayland-protocols";
+  version = "1.6.0-deepin.1.2";
```

Comment:
It can't be used in `src.rev`?

Reply:
dde-kwin/deepin-kwin/deepin-wayland-protocols/dwayland use snipe branch（for deepin v23)，This branch has just been synced to github, but the tag is not synced. so use commit like `chore: Update version number for 5.24.3-deepin.1.9` https://github.com/linuxdeepin/dwayland/commit/3e2d16d4278959c2aa76e1e1f4c0cff9f52a109f

upd:  now the tags of other projects have been synchronized except deepin-kwin